build.gradle

```diff
@@ -55,6 +55,8 @@ dependencies {
   implementation 'commons-io:commons-io:2.11.0'
   implementation "commons-beanutils:commons-beanutils:1.9.4"
   implementation "org.apache.commons:commons-lang3:3.12.0"
+  implementation 'org.json:json:20220924'
```

Comment:
Could we used `ObjectMapper` and `JsonNode` instead to avoid having to bundle another dependency?

e.g.
```java
ObjectMapper objectMapper = new ObjectMapper();
JsonNode jsonNode = objectMapper.readTree(strOriginalContent);

for (Iterator<String> fieldIterator = jsonNode.fieldNames(); fieldIterator.hasNext();) {
  String fieldName = fieldIterator.next();
  
  if (!fixedFields.contains(fieldName)) {
    fieldIterator.remove();
  }
}
```

Reply:
It is updated to replace org.json. Thanks!